helper/dynamic_property_support.go

```diff
@@ -0,0 +1,85 @@
+package helper
+
+import (
+	"fmt"
+	"github.com/buildpacks/libcnb"
+	"github.com/magiconair/properties"
+	"github.com/paketo-buildpacks/libpak/bard"
+	"os"
+	"strings"
+)
+
+const prefix = "BPL_TOMCAT_ENV_"
+
+type DynamicPropertySupport struct {
+	Bindings libcnb.Bindings
+	Logger   bard.Logger
+}
+
+func (a DynamicPropertySupport) Execute() (map[string]string, error) {
+	systemProperties := make(map[string]string)
+
+	for _, b := range a.Bindings {
+		a.Logger.Infof("Configuring properties for binding %s", b.Name)
+
+		for k, v := range b.Secret {
+			systemProperties[fmt.Sprintf("%s.%s", b.Name, k)] = v
+		}
```

Comment:
I think we are assuming that the binding will be one-to-one with one binding providing one value, but that could get tedious. Do you think we should support a single binding providing multiple keys?

Maybe we could support two binding types? i.e. `tomcat-properties` and `tomcat-properties-file`? Where the former is the one-to-one scenario and the latter would be treated as each key provides a set of key=value properties to set?